modules/doc/content/getting_started/installation/moose_conda_binary.md

```diff
@@ -0,0 +1,153 @@
+many available physics solvers, you can install the pre-built fully-featured MOOSE binary.
+!style-end!
+
+## Install Conda
+
+!style! halign=left
+Our preferred method for deliverying pre-built MOOSE binaries is via Conda.
+!style-end!
+
+!include getting_started/installation/install_miniconda.md
+
+## Install MOOSE
+
+!style! halign=left
+With Conda initialized, create the `moose` environment and install `moose`:
+!style-end!
+
+```bash
+mamba create -n moose moose
+```
+
+After the installation completes, activate the new environment:
+
+```bash
+mamba activate moose
+```
+
+!alert note
+Know that you will need to `mamba activate moose` for +each terminal window you open, and each time
+you wish to perform MOOSE related work+. If you wish to make this automatic, you can add that
+command to the end of your shell profile.
+
+Verify `moose` is available:
+
+```bash
+moose --help
+```
+
+Should produce the help page. This simple command demonstrates that you have sucessfully installed
+the MOOSE Conda package.
+
+## Run an Example
+
+!style! halign=left
+A MOOSE installation binary comes with several examples you can run to make sure everything
+is sound, as well as moving some of the example inputs into a safe location you can play with.
+!style-end!
+
+There are examples for each physics solver available by name, in the following directory:
+
+```bash
+ls $CONDA_PREFIX/moose/share/combined
+```
+
+!alert! note
+Not everything you find in this directory is a physics library. We are working on an elegant way to
+ask `moose` for all available solvers.
+!alert-end!
+
+For now, lets copy the reactor module into a safe location for editing:
+
+```bash
+mkdir -p ~/projects/examples
+cd ~/projects/examples
+moose --copy-inputs reactor_workshop
+```
+
+!alert! note
+Take note of the information being displayed in the output. `moose` is alerting to the directory
+structure it created. Which can sometimes not represent the exact wordage you provided as arguments.
+!alert-end!
+
+With the reactor module's examples and inputs copied, move into the reactor workshop directory and
+instruct `moose` to run the tests:
+
+```bash
+cd combined/reactor_workshop
+moose --run -j 6
+```
+
+Testing will commence and take a few moments to finish. There may be several skipped tests for one
+reason or another. This is normal. However no test should fail.
+
+Next, we will run a single test manually, to demonstrate how you will ultimately be using `moose`.
+Navigate to the following directory, and run the following input file:
+
+```bash
+cd ~/projects/examples/combined/reactor_workshop/tests/reactor_examples/abtr/
+moose -i abtr.i --mesh-only --error-deprecated
+```
+
+You will see some information scroll by, and ultimately end back up at your prompt. If you perform a
+direcotry listing you should also see that an exodus file was generated in the process
+(`abtr_in.e`):
+
+```bash
+ls
+abtr_griffin_snippet.i  abtr.i  abtr_in.e
+```
+
+## Viewing Results
+
```

Comment:
we need a note that we must have a X window or something to do this?

Reply:
Nope